Add instructions on how to build from a local Docker image

Kobzol opened the pull request:
I have often needed to build a Apptainer container from a local Docker image (not from a remote image in a Docker registry). I think that this might be a relatively popular use-case, but currently this workaround is only described in an issue on GitHub (https://github.com/apptainer/singularity/issues/1537#issuecomment-557527638).

It would be nice to also have it in the official docs.

WestleyK commented on installation.rst:
```diff
@@ -58,7 +58,8 @@ similar with other package managers.
         pkg-config \
-        git
+        git \
+        cryptsetup-bin
```
I suggest using `cryptsetup`, because `cryptsetup-bin` looks to be a dependency for `cryptsetup`

```suggestion
        cryptsetup
```

Kobzol committed:
Add instructions on how to build from a local Docker image

I have often needed to build a Apptainer container from a local Docker image (not from a remote image in a Docker registry). I think that this might be a relatively popular use-case, but currently this workaround is only described in an issue on GitHub (https://github.com/apptainer/singularity/issues/1537#issuecomment-557527638).

It would be nice to also have it in the official docs.
